src/sentry/integrations/github/issues.py

```diff
@@ -229,11 +235,14 @@ def get_link_issue_config(self, group: Group, **kwargs: Any) -> list[dict[str, A
             {
                 "name": "comment",
                 "label": "Comment",
-                "default": "Sentry issue: [{issue_id}]({url})".format(
-                    url=absolute_uri(
-                        group.get_absolute_url(params={"referrer": "github_integration"})
+                "default": (
+                    get_linked_issue_comment_prefix(group)
+                    + ": [{issue_id}]({url})".format(
+                        url=absolute_uri(
+                            group.get_absolute_url(params={"referrer": "github_integration"})
+                        ),
+                        issue_id=group.qualified_short_id,
```

Comment:
didn't get a chance to comment this before it was merged, but this line is too long, and is doing too many things IMO -- 
I would suggest moving this to a function, and attempting to make it more readable.

https://refactoring.com/catalog/extractVariable.html